### DISCUSSION TITLE
svga能转成PAG文件吗？

### DISCUSSION THREAD

**Discussion by tang9716** [Tencent/libpag#1727] [General] [answered]
svga能转成PAG文件吗？

**Comment by kevingpqi123** [comment, marked as answer]
svga 不能直接转 pag 文件，需要找到 svga 文件的 AE 工程，在 AE 工程中选择导出  pag 文件

**Comment by tang9716** [comment]
svga转图片帧，图片帧转pag可以吧

**Reply by domchen** [reply]
这样可以的

**Reply by ruanchuanli** [reply]
> svga转图片帧，图片帧转pag可以吧

这个怎么实现呢，根据pag格式规范文档，自己写脚本吗

**Reply by kevingpqi123** [reply]
此类问题到 pag 官网上找， https://pag.art/docs/SDK-migration.html
